### PR TITLE
Added Deployment box/instance requirements to README.

### PR DESCRIPTION
**Pull request by ccjoel**
Any other suggestions on what we may add to make it more friendly for others to deploy?

### CODE REVIEW

**Comment by ccjoel** [comment]
Any additional gotchas you ran into when setting recently @david-andrew?

Even if locally, we can add some more details to README or elsewhere.

**Comment by david-andrew** [comment]
@ccjoel local setup is pretty trivial. You just need to do:
- make a new python environment
- pip install requirements.txt
- set the environment variable OPENAI_API_KEY

File: README.md [hunk 1]
```diff
@@ -53,3 +53,14 @@ When `make up` command is run, the Ubuntu image is pulled and loaded in to the i
 If you need a different base image loaded, you can load it with this command: `docker-compose exec docker docker pull jataware/dojo-publish:{base_image_tag_name}`
 
 Since the Docker service has a persistent volume, you should not need to rerun the command unless changes have been made to the image.
+
+# Deployment
+
+## Recommended System Requirements
+
+Due to newer features such, as semantic search, we recommended:
```
Comment (adamgilg) [inline README.md]: typo here with the comma: should be "Due to newer features, such as semantic search, we recommend:"

File: README.md [hunk 2]
```diff
@@ -53,3 +53,14 @@ When `make up` command is run, the Ubuntu image is pulled and loaded in to the i
...
+
+- 6-8 CPU Cores
+- 32GB RAM (memory)
+
+This matches, and has been tested on, an `t3.2xlarge` instance on `AWS`.
```
Comment (adamgilg) [inline README.md]: While you're changing the above, this should be "a `t3.2xlarge`" instead of "an"

**Review by adamgilg — APPROVED** [review]

**Comment by adamgilg** [comment]
I wrote comments about two extremely minor typos, but then I decided I should just fix them. Not sure if it needs more details, but what's there looks good.

**Comment by ccjoel** [comment]
Thanks @adamgilg . I appreciate that review. I suppose I should have caught the wrong coma placement.

@david-andrew these are dojo readme changes, not boxytas. Not sure if you were referring to boxytas itself.

**Comment by ccjoel** [comment]
I'm merging this in the meantime while we think of other dev docs to add to dojo later on.

**Pull request merged**